### ISSUE TITLE
Bokeh Panel class gone in v3.0.3

### ISSUE THREAD

**Issue by rickecon** [PSLmodels/Cost-of-Capital-Calculator#374]
This doesn't need to be updated now. But I wanted to post it as you think about updating to allow Bokeh v3.0.3.

I was running CCC code from a conda environment with Bokeh v3.0.3 and I got the following import failure:
```
ImportError: cannot import name 'Panel' from 'bokeh.models.widgets' (/Users/richardevans/opt/anaconda3/envs/tn-bustax-dev/lib/python3.10/site-packages/bokeh/models/widgets/__init__.py)
```
This [StackOverflow thread](https://stackoverflow.com/questions/75016142/did-the-panel-class-move-to-another-library-in-bokeh-for-new-version-of-bokeh) suggests that the `bokeh.models.widgets.Panel` class has been replaced in Bokeh v3.0.3 with `bokeh.model.TabPanel` (see [Bokeh 3.0.3 documentation](https://docs.bokeh.org/en/latest/docs/reference/models/layouts.html#bokeh.models.TabPanel)). This looks like the whole `widgets` section has been moved/removed.

I noted that the `Panel` and `Tabs` classes only appear in CCC in `calculator.py`

cc: @jdebacker

**Comment by jdebacker** [comment]
Related to Issue #354

**Comment by jdebacker** [comment]
`bokeh.models.widgets` is available in the latest release of Bokeh (3.2.2) (see [docs here](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.html))
